.gitignore

```diff
@@ -11,3 +11,4 @@ PHP_BrowscapINI
 ergebnis.txt
 tests/data/*
 !tests/data/.gitkeep
+.php_cs.cache
```

Comment:
You added this here, but I don't see the `setUsingCache` config set in the `.php_cs` file. Was php-cs-fixer creating the cache file even with that disabled (maybe it's default now)?

Thoughts on enabling and using the cache?  Generally I like having the cache as it speeds up the php-cs-fixer run considerably.